app/api/leaderboard/functions.ts

```diff
@@ -70,62 +70,57 @@ export default async function fetchGitHubReleases(
   if (!accessToken) {
     return [];
   }
-
-  const response = await fetch("https://api.github.com/graphql", {
-    method: "POST",
-    headers: {
-      Authorization: `Bearer ${accessToken}`,
-      "Content-Type": "application/json",
-    },
-    body: JSON.stringify({
-      query: `{
-        organization(login: "${env.NEXT_PUBLIC_GITHUB_ORG}") {
-          repositories(first: 100) {
-            nodes {
-              name
-              releases(first: 10, orderBy: {field: CREATED_AT, direction: DESC}) {
-                nodes {
-                  name
-                  createdAt
-                  description
-                  url
-                  author {
-                    login
-                    avatarUrl
-                  }
-                  mentions (first: 10) {
-                    nodes {
-                      login 
+  try {
+    const response: ReleasesResponse = await octokit.graphql({
+      query: `
+        query GetReleases($org: String!) {
+          organization(login: $org) {
+            repositories(first: 100) {
+              nodes {
+                name
+                releases(first: 10, orderBy: {field: CREATED_AT, direction: DESC}) {
+                  nodes {
+                    name
+                    createdAt
+                    description
+                    url
+                    author {
+                      login
                       avatarUrl
                     }
+                    mentions (first: 10) {
+                      nodes {
+                        login
+                        avatarUrl
+                      }
+                    }
                   }
                 }
               }
             }
           }
         }
-      }`,
-    }),
-  });
-
-  if (!response.ok) {
-    throw new Error(`HTTP error! status: ${response.status}`);
-  }
-
-  const json = (await response.json()) as ReleasesResponse;
-  const repositories: Repository[] = json.data.organization.repositories.nodes;
-  const allReleases: Release[] = [];
-
-  for (const repository of repositories) {
-    for (const release of repository.releases.nodes) {
-      release.repository = repository.name;
-      allReleases.push(release);
-    }
+      `,
+      org: env.NEXT_PUBLIC_GITHUB_ORG,
+      headers: {
+        Authorization: `Bearer ${accessToken}`,
+        "Content-Type": "application/json",
+      },
```

Comment:
this is not required when using the pre-authenticated octokit instance since the credentials are already specified in the octokit construction